eth2near/eth2near-block-relay-rs/src/eth2near_relay.rs

```diff
@@ -359,7 +359,18 @@ impl Eth2NearRelay {
         let mut headers: Vec<BlockHeader> = vec![];
         let mut current_slot = start_slot;
 
-        while headers.len() < self.max_submitted_headers as usize
+        let remaining_headers = (self
+            .eth_client_contract
+            .get_max_submitted_blocks_by_account()?
```

Comment:
The `get_max_submitted_blocks_by_account` could be cached and requested just once on startup

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge/pull/828/commits/95b9946c7eb0e79fe3fdc5adb934a3f979af61ae